### PR TITLE
Add maintained action

### PR DESCRIPTION
**Pull request by h0x0er**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+<hr/>
+
+## Usage
+
+```yaml
+- name: Discord notification
+  env:
+    DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+  uses: step-security/action-discord@main
```
Comment (varunsh-coder) [inline README.md:21]: this should be a tag and not main

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+  with:
+    args: "The project {{ EVENT_PAYLOAD.repository.full_name }} has been deployed."
+```
+
+**NOTICE :** for stability purposes, it is recommended to use the action with an explicit commit SHA-1 :
```
Comment (varunsh-coder) [inline README.md:26]: we can remove this notice about sha-1

File: README.md (hunk 1 above)
Comment (h0x0er) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (h0x0er) [reply README.md]: Done

**Review by varunsh-coder — APPROVED** [review]

**Review by shubham-stepsecurity — APPROVED** [review]

**Review by varunsh-coder — APPROVED** [review]

**Pull request merged**